public/script.js

```diff
@@ -40,28 +38,55 @@ function getSnippet() {
             } else if (data === "") {
                 alert('Unknown  error has occurred.')
             } else {
-                // sending data and speaking .
+                paragraph.innerHTML = data;
                 speech.text = data;
-                window.speechSynthesis.speak(speech)
-                paragraph.innerHTML = data
+
+                if (clicked) {
+                    window.speechSynthesis.speak(speech);
+                } else {}
```

Comment:
If "else" is empty, you don't need this "else". "else" isn't a requirement in the "if" statement